test/integration/discovery.v1.test.js

```diff
@@ -418,7 +414,6 @@ describe('discovery v1 integration', () => {
       expect(Array.isArray(result.aggregations)).toBe(true);
       expect(result.aggregations[0].results).toBeDefined();
       expect(Array.isArray(result.aggregations[0].results)).toBe(true);
-      expect(result.aggregations[0].results[0].event_rate).toBeDefined();
```

Comment:
why is `event_rate` removed?

Reply:
event_rate is an optional property and is null in this case